### ISSUE TITLE
About LGPN-Ling English and French updated Versions to set online

### ISSUE THREAD

**Issue by Hippodamie** [eXistSolutions/LGPN-ling#422]
Dear Magdalena,

Here at the 2 documents updated versions to set online.
Could you, please, do it?

Many thanks in advance,
with all best wishes,
Sophie

[About LGPN-Ling 24 avril 2018.docx](https://github.com/eXistSolutions/LGPN-ling/files/1944282/About.LGPN-Ling.24.avril.2018.docx)
[About LGPN-Ling 24 avril 2018.pdf](https://github.com/eXistSolutions/LGPN-ling/files/1944283/About.LGPN-Ling.24.avril.2018.pdf)
[Sur LGPN-Ling 24 avril 2018.docx](https://github.com/eXistSolutions/LGPN-ling/files/1944284/Sur.LGPN-Ling.24.avril.2018.docx)
[Sur LGPN-Ling 24 avril 2018.pdf](https://github.com/eXistSolutions/LGPN-ling/files/1944285/Sur.LGPN-Ling.24.avril.2018.pdf)

**Comment by plutonik-a** [comment]
@Hippodamie wrote:

> Hi, Alexandra,
> are missing in the current state of Foreword :
> β
> 1)
> 2)
> 3)
> 4)
> c.
> d.
> of the revisioned version.
> And all that follows seems still to be the former version. Since we have almost re-written/updated every §, could you please make the substitution for each of them? 

---

As I mentioned in my comment above, this issue is in progress. I've added a label "in progress" to this issue, so you can exactly see, what I'm currently working on.  

What you see on the test server is always a work in progress - I gave you the url for the editor to approve the state for this page only.   

The "about" page is halfway done in English on my local machine - I'm going to update it on the test url as soon as I need your proof reading and final approval before updating your production server with it.  

>  Could you also please keep the bold as in the word and pdf revisioned version ? 

I surely will. I'm keeping the font-style like italics as well as the font-weight like in your doc file. 

> And it would look nicer if the end of lines on the right of the page was aligned as it is on the left side.

But I don't quite understand the right alignment request:  
I haven't changed anything in the left or right alignment, so the layout of the text will align as before.

**Comment by Hippodamie** [comment]
Sorry, Alexandra ! I did not understand ! Now, yes !

OK for all, many thanks.

For the right aligment, see screenshot:

her is my pdf version:


Alignment at right and at left margins.


Here is your in progress version as I see it on my computer. On the right margin, last words on each lines are not aligned: thus it does not look as clean on the right as on the left margin, do you understand what I mean ? Look especially at lines 4, 5 and 8. For us, it is important it looks exactly as the page of a book.

**Comment by plutonik-a** [comment]
@Hippodamie   

> For us, it is important it looks exactly as the page of a book.

Ok, that is a totally new requirement for me. I thought, I was just supposed to update the text, stick to the former web-layout and keep the font-weight and font-style of your word template.

1. About the alignment: What you want is a justified alignment of the text. I get it now.
1. So what about the images attached to the headlines? They were added for the web page, but don't appear in the PDF. Do you want to still keep them? 
1.  Also, the different indentations of the first lines in each chapter/subchapter haven't been considered yet. So far, I sticked to a usual web layout and convenient screen reading.

**Comment by Hippodamie** [comment]
OK
1. yes
2. You can keep the images as they are (Magdalena had added them).
3. OK: I hope it will look fine.

PLease, tell me, when you are the 3rd page of my pdf, so that I have a look; and please give me the link again.

Best,
S.

**Comment by plutonik-a** [comment]
@Hippodamie  

Here's the result of the first two PDF pages:  
http://admin.existsolutions.com:55221/exist/apps/lgpn-ling/about.html?lang=en 

![01-lgpn-ling-about-en](https://user-images.githubusercontent.com/1157669/39367931-7a05b7fc-4a38-11e8-9873-17f4104c9ecb.png)

**Comment by Hippodamie** [comment]
WONDERFUL, Alexandra : lots of thanks !
Best,
Sophie

**Comment by plutonik-a** [comment]
@Hippodamie  Great to hear, that you like the web output! I'm continuously  going to update the test server, so you can always see, what the current progress of my work is. 

The test URL will stay http://admin.existsolutions.com:55221/exist/apps/lgpn-ling/about.html?lang=en for the English translation.

**Comment by Hippodamie** [comment]
Hi, Alexandra.

Great ! It looks beautiful, and the table too : many, many thanks !

Apologies for that...:
could you, please, change Flect 1 (7.3, 3 occurrences + title of the table) into Inflect1.

Many thanks in advance,
best,
Sophie



P.S.:  Here is the character IFAO Greek Unicode you asked for in the previous email, but will you be able to see it ??? -,  -
Here is the link to upload the font:
http://www.ifao.egnet.net/publications/publier/outils-ed/polices/


,

**Comment by tuurma** [comment]
Dear Sophie, 

for the future we'd need to rethink how we proceed with changes to these documents as setting them in HTML from PDF in two language versions is a very lengthy process, taking many hours of time. For minor changes in the future we'd be happy to have maybe screenshots annotated even with pencil but for major reworks we'd suggest you using HTML editor, e.g. with oXygen (see example of OXygen Author mode below)

![image](https://user-images.githubusercontent.com/449468/39421380-0e82c262-4c69-11e8-887b-b6e6480f54d0.png)

**Comment by tuurma** [comment]
Just to illustrate how much markup is required only for this short fragment

```xml
                            <ol>
                                <li>
                                    masc. in -ᾱ/ης or -ος <i>vs</i> fem. in -ᾱ/η, -ᾰ or -ον, e.g. Habragoras/Ἁβραγόρας <i>vs</i> Ηabragora/Ἁβραγόρα,
                                    Kallinikos/Καλλίνικος <i>vs</i> Kallinike/Καλλινίκη, Habryllos/Ἅβρυλλος <i>vs</i> Habrylla/Ἅβρυλλᾰ and Onesiphoros/Ὀνησίφορος
                                    <i>vs</i> Onesiphoron/Ὀνησίφορον, or even the triplet Anchios/Ἄγχιος <i>vs</i> Anchias/Ἀγχίας <i>vs</i> Anchia/Ἀγχία.
                                </li>
                                <li>
                                    fem. in -εια <i>vs</i> masc. in -ύς e.g. Hēdeia/Ἥδεια <i>vs</i> Hēdus/Ἥδυς or <i>vs</i> masc. in -ης, like Eugeneia/Εὐγένεια <i>vs</i> Eugenes/Εὐγένης;
                                </li>
                                <li>
                                    fem. in -ίς <i>vs</i>. masc. in -ις, such as fem. AnaxisἈναξίς <i>vs</i> masc. Ἄναξις;
                                </li>
                                <li>
                                    fem. participles in -ουσα <i>vs</i> masc. -ων, such as Areskousa/Ἀρέσκουσα <i>vs</i> Areskôn/Ἀρέσκων, and
                                    in -ασα <i>vs</i> masc. -ας, like Arkesasa/Ἀρκέσασα <i>vs</i> Arkesas/Ἀρκέσας;
                                </li>
                                <li>
                                    em. in -ώ and -οῦς <i>vs</i> masc. in -ων and -οῦς, e.g. Agathô/Ἀγαθώ and Agathous/Ἀγαθοῦς <i>vs</i> Agathôn/Ἀγάθων
                                    (cf. Ἀριστώ, daughter of Ἀρίστων, at Milet, first century, and see L. Dubois, in <i>Onomatologos</i>, 402, 407 and 418);
                                </li>
                                <li>
                                    compounds whose masc. second element is -άναξ <i>vs</i> fem. -άνασσα, e.g. Archianax/Ἀρχιάναξ <i>vs</i> Archianassa/Ἀρχιάνασσα.
                                </li>
                            </ol>
                        </li>
```

![image](https://user-images.githubusercontent.com/449468/39421444-5b23b6c6-4c69-11e8-96b5-ba63c6460ea6.png)

**Comment by Hippodamie** [comment]
Hi, Magdalena,

Apologies for that, to convey to Alexandra too. And many thanks for the perfect job she is doing.
Be sure we will not do it next time. And be sure such a reworking will not be necessary another time, because we will just have a few adjustements/changes to susbtitute: and so we would do as you tell us, I mean telling the precise line to substitute. I think the ‘About LGPN-Ling’ will not have to change before 2 or 3 years.

with all best wishes,
Sophie

**Comment by tuurma** [comment]
Thanks Sophie!

**Comment by plutonik-a** [comment]
Hello @Hippodamie, 
thanks for the link to link to IFAOGreekUnicode font. I noticed already, that you used it in your PDF. 

I have imported this font to the project/repo now (there doesn't seem to be a copyright problem and its free to use) and applied the font only to all the Greek characters. 

Unfortunately I have to apply it on each occurrence with a css class, which is a little 'expensive' and intricate, but now we are sure that all special Greek characters are covered and displayed - also the one I was missing is displayed perfectly now! - and furthermore, the font is really beautiful for the Greek letters, which PT Sans and all others we used so far weren't.
But it's worth the effort.

* Headlines and normal body text will be displayed as before in `PT Sans`, a sans-sérif font-family.
* Greek characters will be displayed in `IFAOGreekUnicode`.

I will give notice when I update the test server.

//cc @tuurma

**Comment by Hippodamie** [comment]
Hi, Alexandra,

IFAOGreekUnicode is for free use. 
Many, many thanks for your hudge effort! But it already looks beautiful,

with all best wishes,
Sophie

**Comment by plutonik-a** [comment]
Current status of lgpn-ling v1.3.5:  
* The table is completed.
* Greek characters on `about.html` are displayed with IFAOGreekUnicode

Test url: http://admin.existsolutions.com:55221/exist/apps/lgpn-ling/about.html?lang=en

**Comment by Hippodamie** [comment]
Many thanks, Alexandra : it looks beautiful !

Could you, please, change Flect 1 (see 7.3.) and title of the table into Inflect1 ANF also in the title Flectional Morphems into Inflectional ?
Sorry for this last change.
Many thanks in advance,
best,
Sophie

**Comment by plutonik-a** [comment]
Current status of lgpn-ling v1.3.7:  
* Change occurrences of "Flect 1" in chapter 7.3 to "Inflect 1"
* Change table title to "Inflectional Morphemes (Inflect1) in the nominative and genitive"

Test url: http://admin.existsolutions.com:55221/exist/apps/lgpn-ling/about.html?lang=en

**Comment by Hippodamie** [comment]
Great, Alexandra : many, many thanks !
Best,
Sophie

**Comment by Hippodamie** [comment]
Hi, Alexandra,

Sorry, for this addition I ask, which was just sent to me yesterday by one of the project coordinators. 

Here is the change I ask for, in Foreword:

TITLE TO CHANGE
           d. Typology of compounds. 

INTO
d. Compounds.
α) Typology.

§ ALREADY DONE BY YOU. TO KEEP SUCH.Given the overrepresentation of Hellenistic and Imperial material, the classification system adopted for compounds has been mainly synchronic (headedness, XYZ labels) rather than resting on inherited syntactic categories (possessives, hypostatics, verbal rection, determinatives)[iv] <applewebdata://0DC01599-1B99-4ED9-9D24-F691A647851A#_edn4>.

Systematic treatment of anthroponymic onomastics viewed as a lexicon personae is a all the more sensitive matter as this field has been even more creative than for the lexicon at large, especially regarding name compounding and derivation of abbreviated compounds and simplicia. Thus, the logical labelling of structures reshaped by analogical renovation within particular series of compounds has often to remain hypothetical. For instance, regarding compounds whose second element is a verbal adjective in –τος, some were probably coined on the analogy of a reverse structure having a first verbal element in –σι, such as Lysippos/Λύσ-ιππος vs Hippolytos/Ἱππό-λυτος, whence YX<XY, see 8 and n. 5 below[v] <applewebdata://0DC01599-1B99-4ED9-9D24-F691A647851A#_edn5> . The same holds true for names like archaic Aeol. Abanthis/Ἀβ-ανθίς (<*αυσ-ανθ-, with οὖς ‛ear’ <*aus[vi] <applewebdata://0DC01599-1B99-4ED9-9D24-F691A647851A#_edn6> and ἄνθος ‘flower’), YX(?) vs reverse Cret. Anthôtas/Ἀνθ-ώτας, XY/YX, which are all the more opaque for us as they remain isolated. § ALREADY DONE BY YOU. TO KEEP SUCH.



HERE IS THE ADDITION

β) Etymological segmentation
Segmentation of compound names is systematic: our aim is to highlight their etymology, even if they are made of converted words of the Lexicon, as for Κάτ-αρχος, deverbative of the compound κατ-άρχω ‘begin, command’, whose derivative agent noun is κατάρχης ‘founder, domineering’. Since the compound's meaning may be quite different of the etymological meaning of each of its elements, as for Ὕπ-αρχος ‘under, somewhat, secretly’ + ‘leader, first magistrate, beginner’ vs. ὑπάρχω ‘begin, lay down’ and its agent noun ὕπαρχος ‘subordinate commander’, the translation of the corresponding or coincidental noun is then given in French in Lexis (cf. 10), where a Logeion link gives access to the English and sometimes Spanish translations.



Here is the same § (bis), without any yellow colour:
β) Etymological segmentation

Segmentation of compound names is systematic: our aim is to highlight their etymology, even if they are made of converted words of the Lexicon, as for Κάτ-αρχος, deverbative of the compound κατ-άρχω ‘begin, command’, whose derivative agent noun is κατάρχης ‘founder, domineering’. Since the compound's meaning may be quite different of the etymological meaning of each of its elements, as for Ὕπ-αρχος ‘under, somewhat, secretly’ + ‘leader, first magistrate, beginner’ vs. ὑπάρχω ‘begin, lay down’ and its agent noun ὕπαρχος ‘subordinate commander’, the translation of the corresponding or coincidental noun is then given in French in Lexis (cf. 10), where a Logeion link gives access to the English and sometimes Spanish translations.



Could you please add it ?

Many thanks in advance,

Best,
Sophie

**Comment by Hippodamie** [comment]
Dear Alexandra,

I give you a Powerpoint view where are attached all the sponsors logos we need to attach at the end of the About LGPN-Ling text, where is the copyright and the up-dated date like May 2018.

Many thanks in advance.

with all best wishes,
Sophie

**Comment by plutonik-a** [comment]
Hello @Hippodamie  
In your previous post you mentioned an attachment, which I would like to insert, but apparently nothing has been attached to the Gitlab Post here.  
Could you please upload the assets again?  

In fact, I need to get the original (at best high resolution PNG/JPG or SVG) images and graphics from you, so if it's all extractable from the powerpoint - brilliant (otherwise please provide links to the websites, where I can copy the images from).

**Comment by Hippodamie** [comment]
Dear Alexandra,

The others : I do not have anything else.
They have to be ordered exactly as in the attached PPT.
They are exactly 8 Logos, as in the PPT.


        
I do not know the adresses where we find the images in each institution, I just can give you the adresses of the institutions themselves….

https://www.psl.eu
https://www.ephe.fr
http://www.lgpn.ox.ac.uk/names/LGPN-Ling.html
http://www.sorbonne-universite.fr
http://www.anhima.fr
http://www.ox.ac.uk

Best,
Sophie





http://www.sorbonne-universite.fr
http://www.anhima.fr
http://www.ox.ac.uk

**Comment by plutonik-a** [comment]
Dear @Hippodamie ,  

Thanks for the links to the images!
Unfortunately, I still haven't received the powerpoint slides. How do you attach them?  
Are you communicating via e-mail only? That might not work when attaching assets, because Gitlab might filter them and thus are completely dropped.  

Please attach them directly via Gitlab using the comment box and then selecting the assets there:

<img width="707" alt="gitlab-comment" src="https://user-images.githubusercontent.com/1157669/39923802-7b3f88ee-5525-11e8-80f6-85e7b811f2e3.png">

**Comment by plutonik-a** [comment]
@Hippodamie  

I've updated the requested changes (as posted in https://github.com/eXistSolutions/LGPN-ling/issues/422#issuecomment-386287954) in chapter "d. Compounds" now:

Test url: http://admin.existsolutions.com:55221/exist/apps/lgpn-ling/about.html  

<img width="1027" alt="bildschirmfoto 2018-05-11 um 14 35 33" src="https://user-images.githubusercontent.com/1157669/39924608-c2611744-5528-11e8-8cb0-2f3bb6d51bd2.png">

**Comment by Hippodamie** [comment]
It looks very well: many, many thanks, Alexandra !
have a good week-end,
best,
Sophie

**Comment by plutonik-a** [comment]
Thanks a lot Sophie, you too. 
Can I remind you please to have a look at https://github.com/eXistSolutions/LGPN-ling/issues/422#issuecomment-388346793 again (about the PPT attachments)?  
I will need the PPT earliest on Monday, not today, so no hurry.

**Comment by Hippodamie** [comment]
[Sonsors.pptx](https://github.com/eXistSolutions/LGPN-ling/files/1995859/Sonsors.pptx)


Is it still not OK ???
I did it once before on Git Hub, the same way... I do not understand.

Docx, now... the same slide (the PTT itself has only 1 slide, with the logos).

[Sponsors.docx](https://github.com/eXistSolutions/LGPN-ling/files/1995863/Sponsors.docx)


Do you get it ? If not, what to do ?

The corresponding screenshot:


<img width="865" alt="capture d ecran 2018-05-11 a 17 26 41" src="https://user-images.githubusercontent.com/16923546/39932248-ef563d80-553f-11e8-928a-41606ba9f832.png">


Best,
Sophie

**Comment by plutonik-a** [comment]
Thanks Sophie!
Now, everything has been correctly uploaded to Github. I see the PPT and the screenshot.   
Great, have a nice week-end, too!

**Comment by plutonik-a** [comment]
@Hippodamie  
Just on a side note:  
You've added a screenshot with a strange layout bug in the homepage header image.  
Just for making sure, I've uploaded the homepage to the test server http://admin.existsolutions.com:55221/exist/apps/lgpn-ling-home/index.html, but I can't see an error there.   

May I ask **which browser (+ version) on what system** you are using, so that I can research, if there is a bug with the browser version you use?  

I'll attach the screenshots to show what I mean:

**Your Screenshot:**
<img width="914" alt="layout-bug" src="https://user-images.githubusercontent.com/1157669/39933155-7c6ee8d2-5542-11e8-9b24-732695086fd0.png">

**Original Layout:**
<img width="1280" alt="original-layout" src="https://user-images.githubusercontent.com/1157669/39933128-66e6421c-5542-11e8-9571-a4164b00d2e3.png">

**Comment by Hippodamie** [comment]
Yes, it was on Safari, at this address:
http://clas-lgpn4.classics.ox.ac.uk/exist/apps/lgpn-ling-home/index.html <http://clas-lgpn4.classics.ox.ac.uk/exist/apps/lgpn-ling-home/index.html>
Here is the page I was on:
and below, how LGPN-Ling home page (top) looks like, just as you remarked !!!

What is the problem ?
But on Firefox it looks as it should !

**Comment by plutonik-a** [comment]
Let me recap:  

1. My setup is MacOS + Safari 11.1 // Chrome 66 // Firefox 59 + 60:
2. Layout in URL http://clas-lgpn4.classics.ox.ac.uk/exist/apps/lgpn-ling-home/index.html is ok on all mentioned browsers
3. Layout in test server URL is also ok on all browsers.

What was your setup, where you experienced the bug? I didn't quite understand your explanation. We both seem to work on MacOS, because you mentioned using Safari.   
So your version would be interesting to know. 

If the bug isn't reproducible by you, I would guess, this was just a random hick-up.

**Comment by Hippodamie** [comment]
I am on Safari 10.1 and Firefox 59. The bug was on Safari, maybe because it is a little old version.
It does not matter, I hope !
Have a good week-end,
best,
Sophie

**Comment by plutonik-a** [comment]
## Current status of lgpn-ling v1.3.12

* Content has been completed for English version
* All HTML markup has been formatted with semantically correct elements (I sticked to the original layout of the WORD original though) and can be 
* TODO: 
   1. Insert content for French version
   2. Create sponsor footer for about page: Needs some rework of surrounding page template
   3. Create "jump-to-top" markers after each major section for users to quickly go back to the top navigation

Test url: http://admin.existsolutions.com:55221/exist/apps/lgpn-ling/about.html?lang=en

![02-lgpn-ling-about-en](https://user-images.githubusercontent.com/1157669/39946502-a7912472-556e-11e8-8166-c029023b431e.png)

**Comment by Hippodamie** [comment]
Many thanks, Alexandra.

1. Please, note that date of up-dating has to be changed into May 2018. And do not forget to add my name under the title, as it was in the original document, since I wrote the introduction.

2. I attach a Word document which is the revision of your HTML English version. It was made by Gerard Genevrois and I. There are things to improve and little things missing in comparison of our original version.


Many thanks for the implemantation of our corrections/marking.

with all best wishes,
Sophie

**Comment by Hippodamie** [comment]
[About LGPN (Alexandra) relu GG_2018-05-12.docx](https://github.com/eXistSolutions/LGPN-ling/files/1997933/About.LGPN.Alexandra.relu.GG_2018-05-12.docx)


Hi, Alexandra.
Here is the same Doc Word as in my previous email, to be sure you get it.
Thanks for your help.
With all best wishes,
Sophie

**Comment by Hippodamie** [comment]
Hi, Alexandra,

nice to have met you ! Thanks for the time spent for our talk.
Just to confirm that the By way of example, for Εὐφραῖος..... after b. and before 7.3 is supposed to be in the same indentation as the previous By way of example of a.
Same size, same indentation.
Many thanks in advance.
I do not forget the up-dated French version.
with all best wishes,
Sophie

**Comment by Hippodamie** [comment]
Here is the translation of § 8.2.2.4:

8.2.2.4. Deverbative of lexical verbal compounds: Dev*
       By way of example: Katarchos/Κάταρχος « founder, commander », deverbative of κατάρχω « begin, command », see the other deverbatifve occurring itself in the lexicon, κατάρχης  « founder, dominator » (Pind.).
	Apokatastanōn/Ἀποκαταστάνων « who re-establishes, restores », deverbative of ἀποκαθιστάνω « re-establish, restore » (Plb.), thematised form of ἀποκαθίστημι (Hp.+).

Thanks for your help and work,
with all best wishes, 
Sophie

**Comment by plutonik-a** [comment]
Hello Sophie,

I have got a formatting question about the chapter "Further Reading":
The WORD/PDF original has been formatted with text indents on the first line of each book reference, which looks ok in the printed version.  
In the web output, the width of the page-lines is wider, so that some lines are too short to break into the next line with the result, that the overall layout looks very bumpy.  

I will demonstrate you with following screenshots, what my alternative suggestion for the web would look like - which version do you prefer?

**Original in Print**
  
<img width="704" alt="further-reading-original" src="https://user-images.githubusercontent.com/1157669/40228458-a0c657fc-5a91-11e8-8a50-aa8743bef5ff.png">

**1. Original formatting kept for web output** 
 
<img width="869" alt="further-reading-text-indent" src="https://user-images.githubusercontent.com/1157669/40228551-c9c1e130-5a91-11e8-9027-e76cf96b0ae2.png">

**2. Alternative suggestion: Formatted like a list:** 
 
<img width="844" alt="further-reading-list" src="https://user-images.githubusercontent.com/1157669/40228571-ded410d4-5a91-11e8-802e-79ad72b02a94.png">

**Comment by plutonik-a** [comment]
@Hippodamie 
Another question: 
In chapter 8.1.1 you added a correction for 

<img width="773" alt="bildschirmfoto 2018-05-18 um 14 21 19" src="https://user-images.githubusercontent.com/1157669/40234474-491a1de8-5aa7-11e8-8ef5-36b552680391.png">

I've corrected it into 

<img width="830" alt="bildschirmfoto 2018-05-18 um 14 23 36" src="https://user-images.githubusercontent.com/1157669/40234484-52ada2b2-5aa7-11e8-9f46-e8df29d27bb7.png">

Is this the way you meant it?   
Anyway, on the next occasion, could you please insert your corrections directly in the text and activate the highlighting of corrections in your WORD file? It will be easier for me to just copy and replace the content as it is.

**Comment by plutonik-a** [comment]
Hi @Hippodamie 

I have finished the English content now. Please have a look, if everything is ok on http://admin.existsolutions.com:55221/exist/apps/lgpn-ling/about.html 
and also consider my last 2 posts above.

**Comment by Hippodamie** [comment]
Hello, Alexandra,

Thank you very much.
I prefer the second one, BUT without the dot for the first SAGA.
Many thanks,
best,
Sophie

**Comment by Hippodamie** [comment]
Many thanks for that.
Ok, sorry. I will do it next time.
Have a good week-end,
best,
Sophie

**Comment by plutonik-a** [comment]
@Hippodamie 

> I prefer the second one, BUT without the dot for the first SAGA.

Ok, I've also fixed that now and updated the test server (http://admin.existsolutions.com:55221/exist/apps/lgpn-ling/about.html):  

<img width="877" alt="bildschirmfoto 2018-05-18 um 19 32 50" src="https://user-images.githubusercontent.com/1157669/40249144-5b50cc52-5ad2-11e8-8954-a78759ef72d3.png">

**Comment by Hippodamie** [comment]
Alexandra,

I now will have a careful look on your work but it looks wonderful at first sight !
Bravo, bravissimo !
ET un immense merci !
I come back to you later to tell you if this is totally OK
Best,
Sophie

**Comment by Hippodamie** [comment]
Hi, Alexandra:

Thank you for Further reading. Wonderful the By way of example !
Last corrections on the presentation or small errors 

1.Missing: to add, please, your full name in the note 1, to the name of Magdalena. And the same of course in the French version I already gave you.

2. Could you, please, improve on the whole document the right margin, as you did at the beginning when I asked you for? You did it, but from 7., it is not done anymore, and does not look pretty. Hope it is not too much work. And also for the 14 notes of the end.
I, of course, will ask the same for the French version. Aligment at left and at right. It is more nice to read. 

3. Blanks between the Supertitles (the Working interface etc.) could be smaller, I think.

4. An error: Translation is c. note b. (in Foreword).

5. Bold in 8.1. are missing for the initial letter(s) of N(oun), Adj, Pro Adv. N, Adj, Pro Adv in bold, please.

6. Bold missing in 8.2.2.3.: put XY/YX* in bold. Add an s at the end of Example.

7.  The same in 8.2.2.4.: put in bold Deverbative and Dev*. And change By way of example into: Examples. Thank you!

8. Change the bold into normal in 9, in the example of Archi-anax: no bold anymore for the word shift and the word lord.

9. Blank also a little to big before Further reading.

MANY thanks for all what you did.
Have a good week-end.

best,
Sophie

**Comment by Hippodamie** [comment]
Hi, Alexandra:

Thank you for Further reading. Wonderful the By way of example !

1.Missing: to add, please, your full name in the note 1, to the name of Magdalena. And the same of course in the French version I already gave you.

2. Could you, please, improve on the whole document the right margin, as you did at the beginning when I asked you for? You did it, but from 7., it is not done anymore, and does not look pretty. Hope it is not too much work. And also for the 14 notes of the end.
I, of course, will ask the same for the French version. Aligment at left and at right. It is more nice to read. 

3. Blanks between the Supertitles (the Working interface etc.) could be smaller, I think.

4. An error: Translation is c. note b. (in Foreword).

5. Bold in 8.1. are missing for the initial letter(s) of N(oun), Adj, Pro Adv. N, Adj, Pro Adv in bold, please.

6. Bold missing in 8.2.2.3.: put XY/YX* in bold. Add an s to Example.

7.  The same in 8.2.2.4.: put in bold Deverbative and Dev*. And change By way of example into: Examples. Thank you.

8. Change the bold into normal in 9, the example of Archi-anax: no bold anymore for the word shift and the word lord.

9. Blank also a little to big before Further reading.

MANY thanks for all what you did.
Have a good week-end.
I put it also on GitHUb,
best,
Sophie

**Comment by plutonik-a** [comment]
Checklist of todo's from https://github.com/eXistSolutions/LGPN-ling/issues/422#issuecomment-390305066

- [ ] Add Alex' name next to Magda's in note 1 (en | fr).
- [ ] Fix `text-align: justify` beginning from ch.7 plus the notes
- [ ] Space between the Supertitles (the Working interface etc.) could be smaller
- [ ] Translation is c. note b. (in Foreword).
- [ ] Bold in 8.1. are missing for the initial letter(s) of N(oun), Adj, Pro Adv. N, Adj, Pro Adv in bold
- [ ] Bold missing in 8.2.2.3.: put XY/YX* in bold. Add an s at the end of Example.
- [ ] The same in 8.2.2.4.: put in bold Deverbative and Dev*. And change By way of example into: Examples. 
- [ ] Change the bold into normal in 9, in the example of Archi-anax: no bold anymore for the word shift and the word lord.
- [ ] Space is too big before Further reading.

**Comment by Hippodamie** [comment]
Exactly, thank you Alexandra !
Best,
Sophie

**Comment by plutonik-a** [comment]
All above requests fixed in en version with LGPN-Ling-1.3.19.
